### ISSUE TITLE
Error in namespaceExports while installing PopSV with Devtools

### ISSUE THREAD

**Issue by mkolovic** [jmonlong/PopSV#3]
Attempting to build a Docker image for PopSV, running in to the following, all dependencies should be installed.

```
> devtools::install_github("jmonlong/PopSV", ref="forPaper")
Downloading GitHub repo jmonlong/PopSV@forPaper
Installing PopSV
Skipping 6 packages not available: Biostrings, DNAcopy, GenomeInfoDb, GenomicRanges, IRanges, Rsamtools
'/usr/lib/R/bin/R' --no-site-file --no-environ --no-save --no-restore CMD  \
  INSTALL '/tmp/Rtmp22B1eO/devtools494648dcff87/jmonlong-PopSV-02e074c'  \
  --library='/usr/local/lib/R/site-library' --install-tests 

* installing *source* package 'PopSV' ...
** R
** tests
** preparing package for lazy loading
Warning: replacing previous import by 'data.table::shift' when loading 'PopSV'
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded
Warning: replacing previous import by 'data.table::shift' when loading 'PopSV'
Error in namespaceExport(ns, exports) : 
  undefined exports: qc.samples.summary
Error: loading failed
Execution halted
ERROR: loading failed
* removing '/usr/local/lib/R/site-library/PopSV'
Error: Command failed (1)
```

I suspect that it is because `qc.samples.summary` no longer exists in the `R` folder but is still in the `man` folder. Also, do you know why it says `Skipping 6 packages not available: Biostrings, DNAcopy, GenomeInfoDb, GenomicRanges, IRanges, Rsamtools`? I am sure I have those installed.

**Comment by jmonlong** [comment]
Oh yes, I'm in the middle of cleaning up this branch/releasing. I'm almost done and I'll release the official version 1 later today. 

I don't know if I'll have time to create docker for it, but for sure these _namespace_ problems will be fixed.

I'll let you know. Thanks for the feeback.

**Comment by mkolovic** [comment]
Ok sounds good. I am building a Docker image to deploy at our location, I can push it to Docker Hub if you are interested. Thanks for the support.

**Comment by mkolovic** [comment]
Issue is fixed with v1.0 release.
